spec/models/proposal_spec.rb

```diff
@@ -34,10 +34,10 @@
     it "gives only the first approver when linear" do
       proposal = create(:proposal, :with_serial_approvers)
       approver1, approver2 = proposal.approvers
-      expect(proposal.currently_awaiting_approvers).to eq([approver1])
+      expect(proposal.currently_awaiting_approvers).to contain_exactly(approver1)
 
       proposal.individual_approvals.first.approve!
-      expect(proposal.currently_awaiting_approvers).to eq([approver2])
+      expect(proposal.currently_awaiting_approvers).to contain_exactly(approver2)
```

Comment:
I switched this in case it won't equate an array with an ActiveRecord collection (which I should probably check), and it's more about the contents of the array/collection.